packages/task/src/browser/quick-open-task.ts

```diff
@@ -290,7 +282,10 @@ export class QuickOpenTask implements monaco.quickInput.IQuickAccessDataService
             }];
         }
 
-        this.quickInputService?.showQuickPick(this.items, { placeholder: `Select the ${buildOrTestType} task to run` });
+        this.quickInputService?.showQuickPick(this.items, {
```

Comment:
@esterw1109 there are 6 more instances of `showQuickPick` in this file. Do we need to handle the gear icon there as well or not?

Reply:
No we don't, I have checked all of theme (they open other items - without the gear icon)..